ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioDev.xcscheme

```diff
@@ -125,6 +124,16 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "889CAB5E27D46CD6007F5892"
+            BuildableName = "HelioDev.app"
+            BlueprintName = "HelioDev"
+            ReferencedContainer = "container:Helio.xcodeproj">
+         </BuildableReference>
+      </BuildableProductRunnable>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BuildableProductRunnable` section has been added to the `ProfileAction`. This change seems to be related to the renaming of the app and should not cause any issues if the new names are correct.
```diff
+ <BuildableProductRunnable
+    runnableDebuggingMode = "0">
+    <BuildableReference
+       BuildableIdentifier = "primary"
+       BlueprintIdentifier = "889CAB5E27D46CD6007F5892"
+       BuildableName = "HelioDev.app"
+       BlueprintName = "HelioDev"
+       ReferencedContainer = "container:Helio.xcodeproj">
+    </BuildableReference>
+ </BuildableProductRunnable>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->